src/realtime_ui/trajectory.css

```
@@ -0,0 +1,128 @@
+.conversation { width: 100%; padding: 10px 12px; overflow: hidden; border: 0; border-radius: 8px; background: transparent; cursor: pointer; text-align: left; text-overflow: ellipsis; white-space: nowrap; }
+.conversation:hover { background: #eef0f3; }
+.conversation.active { background: #e6e8ec; }
+.conversation small { display: block; overflow: hidden; margin-top: 2px; color: var(--caption); text-overflow: ellipsis; white-space: nowrap; }
+.app { display: flex; flex: 1; min-width: 0; flex-direction: column; }
+.header { flex: none; height: 58px; padding: 0 28px; border-bottom: 1px solid var(--line); }
+.view-title { display: flex; width: max-content; height: 58px; align-items: center; border-bottom: 2px solid var(--blue); color: var(--blue); font-size: 15px; }
+.toolbar { display: flex; flex: none; height: 42px; align-items: center; padding: 0 10px; border-bottom: 1px solid var(--line); }
+.modes { display: flex; gap: 3px; }
+.modes button { display: inline-flex; height: 29px; align-items: center; gap: 6px; padding: 0 9px; border: 0; border-radius: 5px; background: transparent; color: var(--muted); cursor: pointer; }
+.modes button:hover, .modes button.pressed { border: 0; background: #f0f1f3; color: var(--text); }
+.mode-icon { width: 17px; height: 17px; flex: none; fill: none; stroke: currentColor; stroke-width: 1.5; stroke-linecap: round; stroke-linejoin: round; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)